ACKategories/Base/ViewController.swift

```diff
@@ -16,17 +16,12 @@ extension Base {
     public static var viewControllerMemoryLoggingEnabled: Bool = true
     
     /// Base class for all view controllers
-    open class ViewController<ViewModelType>: UIViewController {
+    open class ViewController: UIViewController {
         
         /// Navigation bar is shown/hidden in viewWillAppear according to this flag
-        var hasNavigationBar: Bool = true
+        open var hasNavigationBar: Bool = true
```

Comment:
Do we really want this property to be overridable? I think it might be enough if it was only public so it can only be set by subclasses 🤔 and well at least the explicit type is not necessary

Reply:
Yes I want it overridable, because why not 😄 I don't want to do new update in half a year from now when I realize I need to override it. And I think there is no reason to strictly disable override for this.